routes/api/auth-router.js

```diff
@@ -142,4 +143,32 @@ authRouter.post('/logout', async (req, res, next) => {
   }
 });
 
+authRouter.patch('/avatars', upload.single('avatar'), async (req, res) => {
```

Comment:
functions shouldn't be in router. Separate it as middleware and controller function, and invoke here

Reply:
previous router functions should be separated too